### ISSUE TITLE
myFullPage.fadingEffect.turnOn() just for slides

### ISSUE THREAD

**Issue by ollyradford** [alvarotrigo/fullPage.js#3589]
I have just purchased the fadingEffect extension and want to use it for slides on all but the last section. I have the following which sort of works:
```
var myFullPage = new fullpage('#fullpage', {
	fadingEffect:'slides',
	onLeave: function(origin, destination, direction) {

		if ( destination.isLast ) {	
					
			myFullPage.fadingEffect.turnOff();

		} else if( origin.isLast ) {
						
			myFullPage.fadingEffect.turnOn();
												
		}

	}
}
```

However `myFullPage.fadingEffect.turnOn()` turns on the effect for sections and slides, rather than following the state of fadingEffect set when the plugin was initialised. I have tried  `myFullPage.fadingEffect.turnOn('slides')` but that does not seem to work.

Any pointers greatly appreciated!

Thanks,
Olly

**Comment by alvarotrigo** [comment]
Ill take a look at it!

**Comment by ollyradford** [comment]
Thanks Alvaro!

**Comment by alvarotrigo** [comment]
Fixed! 😃 

Please check the inbox of the email used for the purchase. You should have gotten a message regarding a new version of the fading effect extension that solves this issue.

I'll close the issue as it has been solved in fading effect version 0.1.0.

**Comment by ollyradford** [comment]
Thanks for the swift fix Alvaro!
